src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTickets {
+    private final List<LottoNumbers> lottoTickets;
+
+    public LottoTickets(final List<LottoNumbers> lottoTickets) {
+        checkNullOrEmpty(lottoTickets);
+        this.lottoTickets = lottoTickets;
+    }
+
+    private void checkNullOrEmpty(final List<LottoNumbers> lottoTickets) {
+        if (Objects.isNull(lottoTickets) || lottoTickets.isEmpty()) {
+            throw new IllegalArgumentException("lottoTickets는 null이거나 empty 일 수 없습니다.");
+        }
+    }
+
+    public Result check(final LottoNumbers winningNumbers) {
+        Result result = new Result();
+        lottoTickets.stream()
+                .map(lottoTicket -> lottoTicket.check(winningNumbers))
+                .forEach(lottoStatus -> result.report(lottoStatus));
```

Comment:
아래와 같이 foreach문을 표현 할 수 있네요 :-) 
```suggestion
                 .forEach(result::report);
```